packages/dev/core/src/Materials/Node/Blocks/Input/prePassTextureBlock.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { NodeMaterialBlock } from "../../nodeMaterialBlock";
+import { NodeMaterialBlockConnectionPointTypes } from "../../Enums/nodeMaterialBlockConnectionPointTypes";
+import type { Effect } from "../../../../Materials/effect";
+import { NodeMaterialConnectionPointDirection, type NodeMaterialConnectionPoint } from "../../nodeMaterialBlockConnectionPoint";
+import type { NodeMaterialBuildState } from "../../nodeMaterialBuildState";
+import { NodeMaterialBlockTargets } from "../../Enums/nodeMaterialBlockTargets";
+import { RegisterClass } from "../../../../Misc/typeStore";
+import type { NodeMaterial } from "../../nodeMaterial";
+import { NodeMaterialConnectionPointCustomObject } from "../../nodeMaterialConnectionPointCustomObject";
+import { Constants } from "core/Engines";
+import type { Mesh } from "core/Meshes";
```

Comment:
those two load from the directory and not from the files themselves. would also be good to stay consistent with lcoal vs. `core` reference, but this is secondary to solving the dependency issue